index.js

```diff
@@ -4,15 +4,37 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = '#088F8F'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.querySelector('div#profile')
+const travelerImg = profile.querySelector('img')
 
+travelerImg.src = traveler.photo
+travelerImg.alt = traveler.name
+
+profile.querySelector('h2').innerHTML = traveler.name
+profile.querySelector('em').innerHTML = traveler.nickname
 
-/***** Deliverable 4 *****/
 
 
+/***** Deliverable 4 *****/
+traveler.animalSightings.forEach(
+    function(sighting) {
+        const unorderedList = document.querySelector('ul#animals')
+        const animalSighting = 
+            `<li data-id="${sighting.id}">
+                <p>${sighting.description}</p>
+                <img src="${sighting.photo}" alt="${sighting.species}"/>
+                <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>
+            </li>`
+        unorderedList.innerHTML += animalSighting
+    }
+)
```

Comment:
Nice! Try the "combo" or "artisanal" approach that we learned in lecture over the `innerHTML +=` approach to make the code more efficient and safer (especially, with the "artisanal" approach)

Reply:
Thank you, Michelle! What changes should I make to achieve the combo approach? I was under the impression that this is the combo approach.